services/src/main/java/org/fao/geonet/api/records/MetadataInsertDeleteApi.java

```diff
@@ -324,6 +324,11 @@ SimpleMetadataProcessingReport insert(
         checkUserProfileToImportMetadata(userSession);
 
 
+        boolean isMdWorkflowEnable = settingManager.getValueAsBool(Settings.METADATA_WORKFLOW_ENABLE);
```

Comment:
A similar change should be applied to https://github.com/geonetwork/core-geonetwork/blob/c6bb9d2cde243700c4773ed372e4311073700c78/services/src/main/java/org/fao/geonet/api/records/MetadataInsertDeleteApi.java#L564